jep/200/README.adoc

```diff
@@ -326,25 +326,25 @@ Some runs of `acceptance-test-harness` (ATH) were already performed in this mode
 `plugin-compat-tester` (PCT) was also run against an array of plugins.
 See the list in _Appendix A_ for more details.
 
-If new whitelist entries are needed after release, they can be added to core in weekly updates.
-Plugins can also contribute their own whitelist (or even blacklist) entries for third-party libraries they bundle,
+If new allowlist entries are needed after release, they can be added to core in weekly updates.
+Plugins can also contribute their own allowlist (or even denylist) entries for third-party libraries they bundle,
 as described in Extensibility above.
 
-Finally, an individual administrator can define site-specific whitelist (or blacklist) entries with a system property `hudson.remoting.ClassFilter`.
-This could be useful as an emergency measure, permitting functionality to be restored while awaiting a new plugin release.
+Finally, an individual administrator can define site-specific allowlist (or denylist) entries with a system property `hudson.remoting.ClassFilter`.
+This could be useful as an emergency measure, allowing functionality to be restored while awaiting a new plugin release.
```

Comment:
why?

Reply:
Just fell into my search and I though "allowed" would sound better. Unrelated to the terminology cleanup